src/core_modules/capture-core/hooks/useTrackedEntityTypesWithCorrelatedPrograms.js

```diff
@@ -32,9 +33,28 @@ export const useTrackedEntityTypesWithCorrelatedPrograms = (): TrackedEntityType
                 },
                 searchGroups,
                 enrollment,
+                stages,
+                useFirstStageDuringRegistration,
```

Comment:
I know that this is the default DHIS2 terminology, but in this case, it really seamed like you were trying to pass a hook as a prop because of the `use`-naming. How does the team feel about this? Could we consider renaming this internally?